Aztec/Classes/TextKit/TextView.swift

```diff
@@ -605,6 +644,37 @@ open class TextView: UITextView {
         return false
     }
 
+    private func shouldRemoveSingleLineParagraphAttributes(insertedText text: String, at location: Int) -> Bool {
+        guard text == String(.newline) else {
+            return false
+        }
+
+        let afterRange = NSRange(location: location, length: 1)
+        var afterString = String(.newline)
+
+        if afterRange.endLocation < storage.length {
+            afterString = storage.attributedSubstring(from: afterRange).string
+        }
+
+        return afterString == String(.newline) && storage.isStartOfNewLine(atLocation: location)
+    }
+
+    @discardableResult func ensureRemovalOfSingleLineParagraphAttributes(insertedText text: String, at range: NSRange) -> Bool {
```

Comment:
I have to admit I'm not able to quicly grasp why this is necessary by just reading the code.

Can you document this method, including information and possibly an example of why this is needed?